art/command_line.py

```diff
@@ -19,7 +19,12 @@
 
 def get_gitlab():
     config = _config.load()
-    return Gitlab(config['gitlab_url'], private_token=config['private_token'])
+    if config['token_type'] == 'private':
+        return Gitlab(config['gitlab_url'], private_token=config['token'])
+    elif config['token_type'] == 'job':
+        return Gitlab(config['gitlab_url'], job_token=config['token'])
```

Comment:
Support for `job_token` was added in `python-gitlab 1.12.0`. We need to update [setup.py](https://python-gitlab.readthedocs.io/en/stable/changelog.html#v1-12-0-2019-10-06) to express this requirement, I think `python-gitlab>=1.12.0` is fine.

I was testing with a `Debian 10` environment, which shipped `1.6.0` and lacked this additional argument.

Reply:
resolved in [ddd8d8d](https://github.com/christheyankee/gitlab-art/commit/ddd8d8d046428f2bb9c71991f361172d15033d0d)